pwnlib/commandline/template.py

```diff
@@ -25,14 +21,9 @@
 parser.add_argument('--color', help='Print the output in color', choices=['never', 'always', 'auto'], default='auto')
 
 def main(args):
-    cache = None
-
-    if cache:
```

Comment:
Wow, so this was dead code, thanks.

Reply:
I think we should just remove the code. If someone complains about the template creation being too slow we could add the caching feature :) What do you think?